go/teams/chain.go

```diff
@@ -79,6 +79,39 @@ func (t TeamSigChainState) GetLatestHighLinkID() keybase1.LinkID {
 	return t.inner.LastHighLinkID
 }
 
+func (t TeamSigChainState) GetHighSkip() (libkb.HighSkip, error) {
+	var linkID libkb.LinkID
+	linkIDStr := t.GetLatestHighLinkID()
+	if linkIDStr != "" {
+		preLinkID, err := libkb.ImportLinkID(linkIDStr)
+		if err != nil {
+			return libkb.HighSkip{}, err
+		}
+		linkID = preLinkID
+	}
+	highSkip := libkb.NewHighSkip(t.GetLatestHighSeqno(), linkID)
+	return highSkip, nil
+}
+
+// If the chain has posted any link and still has a LatestHighSeqno
+// of 0, that means it hasn't been computing high skips yet.
+// In the special case of the first link, we already know what the
+// next high skip should be, so reverification is not needed.
+func (t TeamSigChainState) ReverifyNeededForHighSkips() bool {
+	return t.GetLatestSeqno() > 1 && t.GetLatestHighSeqno() == 0
+}
+
+func (t TeamSigChainState) GetHighSkipIfValid() (*libkb.HighSkip, error) {
+	if !t.ReverifyNeededForHighSkips() {
+		highSkip, err := t.GetHighSkip()
+		if err != nil {
+			return nil, err
+		}
+		return &highSkip, nil
+	}
+	return nil, nil
+}
```

Comment:
For a state at seqno 1 loaded from an old cache, I think this could return default values but it should return seqno=1.